Ошибка: HTTPConnectionPool(host='73', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027AB4B970>: Failed to resolve '73' ([Errno 11001] getaddrinfo failed)"))